MANANPAT-M-VJMD:backup_nh mananpat$ git diff ../../../../experimental/backup_nhg/ate_tests/backup_nhg_test/backup_nhg_test.go
diff --git a/feature/experimental/backup_nhg/ate_tests/backup_nhg_test/backup_nhg_test.go b/feature/experimental/backup_nhg/ate_tests/backup_nhg_test/backup_nhg_test.go
index 480f4010..c124b2a0 100644
--- a/feature/experimental/backup_nhg/ate_tests/backup_nhg_test/backup_nhg_test.go
+++ b/feature/experimental/backup_nhg/ate_tests/backup_nhg_test/backup_nhg_test.go
@@ -152,15 +152,15 @@ func TestDirectBackupNexthopGroup(t *testing.T) {
                        desc: "Disable ATE port-2",
                        applyImpairmentFn: func() {
                                ateP2 := ate.Port(t, "port2")
-                               dutP2 := dut.Port(t, "port2")
+                               // dutP2 := dut.Port(t, "port2")
                                ate.Actions().NewSetPortState().WithPort(ateP2).WithEnabled(false).Send(t)
-                               dut.Telemetry().Interface(dutP2.Name()).OperStatus().Await(t, time.Minute, telemetry.Interface_OperStatus_DOWN)
+                               // dut.Telemetry().Interface(dutP2.Name()).OperStatus().Await(t, time.Minute, telemetry.Interface_OperStatus_DOWN)
                        },
                        removeImpairmentFn: func() {
                                ateP2 := ate.Port(t, "port2")
-                               dutP2 := dut.Port(t, "port2")
+                               // dutP2 := dut.Port(t, "port2")
                                ate.Actions().NewSetPortState().WithPort(ateP2).WithEnabled(true).Send(t)
-                               dut.Telemetry().Interface(dutP2.Name()).OperStatus().Await(t, time.Minute, telemetry.Interface_OperStatus_UP)
+                               // dut.Telemetry().Interface(dutP2.Name()).OperStatus().Await(t, time.Minute, telemetry.Interface_OperStatus_UP)
                        },
                },
                {
@@ -168,17 +168,18 @@ func TestDirectBackupNexthopGroup(t *testing.T) {
                        applyImpairmentFn: func() {
                                dutP2 := dut.Port(t, "port2")
                                dut.Config().Interface(dutP2.Name()).Enabled().Replace(t, false)
-                               dut.Telemetry().Interface(dutP2.Name()).OperStatus().Await(t, time.Minute, telemetry.Interface_OperStatus_DOWN)
+                               // dut.Telemetry().Interface(dutP2.Name()).OperStatus().Await(t, time.Minute, telemetry.Interface_OperStatus_DOWN)
                        },
                        removeImpairmentFn: func() {
                                dutP2 := dut.Port(t, "port2")
                                dut.Config().Interface(dutP2.Name()).Enabled().Replace(t, true)
-                               dut.Telemetry().Interface(dutP2.Name()).OperStatus().Await(t, time.Minute, telemetry.Interface_OperStatus_UP)
+                               // dut.Telemetry().Interface(dutP2.Name()).OperStatus().Await(t, time.Minute, telemetry.Interface_OperStatus_UP)
                        },
                },
        }
        for _, tc := range cases {
                t.Run(tc.desc, func(t *testing.T) {
+                       time.Sleep(20 * time.Second)
                        t.Run("Validate Baseline AFT Telemetry", func(t *testing.T) {
                                tcArgs.validateAftTelemetry(t)
                        })
